install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [[ $1 == "path" && $2 != "" ]]; then
```

Comment:
From what I understand, if you pass this pass value, it will download the teresa binary to the path, right?

What about making it like this:

```sh
./install.sh /my/path
```

That way, you can just test `if [ "$1" ]`.

Reply:
It was just to be more expressive, but if you think it's understandable this way, I can change, no problem :)